src/Symfony/Component/ErrorHandler/Tests/Fixtures/ReturnTypeParent.php

```diff
@@ -200,7 +200,9 @@ public function mixed()
     }
 
     /**
-     * @return mixed|null
+     * @return mixed
+     *
+     * Will be removed in Symfony 6.x, see https://github.com/symfony/symfony/pull/39282 discussion.
```

Comment:
No, it won't. Please restore the previous doc block.

Reply:
But `mixed|null` is NOT valid phpdoc

Reply:
That won't prevent people from using it as you already noticed. This fixture makes sure that we handle this case correctly.

Reply:
I have added `@deprecated` phpdoc, feel free to adjust more if needed

Reply:
All right, I'm closing the PR then.

Reply:
what?

Reply:
what? please reopen, there is a real issue I fixed and spent my time on